lib/applications/field-resolvers.ts

```diff
@@ -23,5 +23,29 @@ export const applicationPermitResolver: Resolver<Application> = async (
   _args,
   { prisma }
 ) => {
-  return await prisma.permit.findMany({ where: { applicationId: parent?.id } });
+  return (await prisma.permit.findMany({ where: { applicationId: parent?.id } }))?.[0];
+};
+
+/**
+ * Field resolver to fetch the application_processing information associated with an application.
+ * @returns ApplicationProcessing object
+ */
+export const applicationApplicationProcessingResolver: Resolver<Application> = async (
+  parent,
+  _args,
+  { prisma }
+) => {
+  return await prisma.applicationProcessing.findUnique({ where: { applicationId: parent?.id } });
```

Comment:
```suggestion
  return await prisma.applicationProcessing.findUnique({ where: { applicationId: parent.id } });
```
Shouldn't need `parent?` here, since the `Resolver<Application>` type enforces that `parent` is of type `Applicant` and is not undefined.